src/Compiler/Checking/MethodCalls.fs

```diff
@@ -352,9 +367,8 @@ let AdjustCalledArgTypeForOptionals (infoReader: InfoReader) ad enforceNullableO
 
                 // If inference has worked out it's a struct (e.g. an int) then use this
                 elif isStructTy g callerArgTy then
-                    let calledArgTy2 = destNullableTy g calledArgTy
-                    AdjustRequiredTypeForTypeDirectedConversions infoReader ad true false calledArgTy2 callerArgTy m
-
+                    AdjustRequiredTypeForTypeDirectedConversions infoReader ad true false calledArgTy callerArgTy m
```

Comment:
I see, so `AdjustRequiredTypeForTypeDirectedConversions` is now doing the work to do the Nullable conversion

Reply:
Right, this keeps all the TDC work in one place, even though it's slightly inconsistently specced. It felt more natural to maintain this case alongside the other conversions.